### PR TITLE
Adds the Dagamuir uniform as a Paramedic uniform option

### PR DESCRIPTION
**Pull request by feartheblackout**
Title/changelog. Approval was acquired prior by @RustingWithYou

### CODE REVIEW

**Comment by feartheblackout** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Review by DreamySkrell — CHANGES_REQUESTED** [review]

File: code/modules/clothing/factions/pmcg.dm [hunk 1]
```diff
@@ -73,6 +73,12 @@
 	icon_state = "pmc_dpf"
 	item_state = "pmc_dpf"
 
+/obj/item/clothing/under/rank/security/pmc/dagamuir_freewater/medical
+	permeability_coefficient = 0.50
+	armor = list(
+		bio = ARMOR_BIO_MINOR
+	)
```
[the remark this answers is not in the document]
Comment (DreamySkrell) [reply code/modules/clothing/factions/pmcg.dm:80]: should not be under /security/ path either

[the remark this answers is not in the document]
Comment (DreamySkrell) [reply code/modules/clothing/factions/pmcg.dm:80]: it should be more like phalanx uniforms, where security and medical phalanx uniforms do differ

[the remark this answers is not in the document]
Comment (feartheblackout) [reply code/modules/clothing/factions/pmcg.dm:80]: Does that include a unique sprite as well in this case?

[the remark this answers is not in the document]
Comment (DreamySkrell) [reply code/modules/clothing/factions/pmcg.dm:80]: yes

**Pull request closed without merging**